cmd/transcoding/transcoding.go

```diff
@@ -42,17 +43,32 @@ func main() {
 	fname := os.Args[1]
 	profiles := str2profs(os.Args[2])
 	accel, lbl := str2accel(os.Args[3])
+	var from, to time.Duration
+	if len(os.Args) > 4 {
```

Comment:
what happened if SW transcoding? So I think it is better to use  flag.Parse().

Reply:
fixed in 9b838127744b93d928de8c9294267caaabb3e599